pkg/linear-pools/contracts/silo-linear-pool/SiloLinearPoolRebalancer.sol

```diff
@@ -43,7 +42,6 @@ contract SiloLinearPoolRebalancer is LinearPoolRebalancer {
     ) LinearPoolRebalancer(ILinearPool(ILastCreatedPoolFactory(msg.sender).getLastCreatedPool()), vault, queries) {
         _shareToken = IShareToken(address(wrappedToken));
         _silo = ISilo(_shareToken.silo());
-        _exchangeRateModel = new SiloExchangeRateModel();
     }
 
     function _wrapTokens(uint256 amount) internal override {
```

Comment:
It would be good to add comments to the wrap/unwrap functions explaining why there is a `false` argument going into deposit/withdraw.